src/core/player.js

```diff
@@ -35,6 +35,33 @@ DM.provide('',
         if (!options || typeof options !== 'object')
             throw new Error("Missing 'options' parameter for DM.player()");
 
+        console.warn('DEPRECATED: Legacy JS SDK integration method is deprecated. Please consider using Player Embeds https://faq.dailymotion.com/hc/en-us/articles/4411096679954-Integrate-your-Player-Embed');
```

Comment:
I would even put the warn before the 2 `throw new Error` ?

no matter the error, we want the warn to be displayed all the time ?

Reply:
👍 done